docs/source/noise/index.rst

```diff
@@ -106,51 +109,51 @@ Noise types for each column
     - Notes
   * - First name
     - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
-    - Leave a field blank, use a fake name, make typos
+    - Leave a field blank, OCR, use a fake name, nickname, make typos, make OCR errors
     -
   * - Middle initial
     - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
-    - Leave a field blank, make typos
+    - Leave a field blank, make typos, make OCR errors
```

Comment:
Note: I haven't checked that these are the right columns to apply the noise type to, but I am assuming you got this from the concept model.